engine/spverrors/definitions.go

```diff
@@ -375,3 +375,18 @@ var ErrBroadcastWrongBUMPResponse = models.SPVError{Message: "broadcast server r
 
 // ErrBroadcastRejectedTransaction is when broadcast server rejected transaction
 var ErrBroadcastRejectedTransaction = models.SPVError{Message: "broadcast rejected transaction", StatusCode: 400, Code: "error-broadcast-rejected-transaction"}
+
+// ErrARCUnreachable is when ARC cannot be requested
+var ErrARCUnreachable = models.SPVError{Message: "ARC cannot be requested", StatusCode: 404, Code: "error-broadcast-unreachable"}
+
+// ErrARCUnauthorized is when ARC returns unauthorized
+var ErrARCUnauthorized = models.SPVError{Message: "ARC returned unauthorized", StatusCode: 401, Code: "error-broadcast-unauthorized"}
+
+// ErrARCParseResponse is when ARC response cannot be parsed
+var ErrARCParseResponse = models.SPVError{Message: "ARC response cannot be parsed", StatusCode: 400, Code: "error-broadcast-parse-response"}
```

Comment:
Here perhaps 500 aswell, as 400 indicates there was something wrong with the request and not the response. WDYT?